Automattic/wp-calypso
# Playwright: migrate theme-activate spec.

#### Changes proposed in this Pull Request

This PR migrates the `wp-theme__activate-spec` from Selenium.

Key changes:

- new method to generate a random integer between bounds in `DataHelper`. This is used to select a random theme if more than 1 matching themes are found after a keyword search.
- new class `ThemeCustomizerPage` to represent the `Appearance > Customize` page.

#### Testing instructions

TeamCity
- ensure that Playwright E2E tests pass.
- ensure that unit tests and other tasks pass.
- ensure that for Playwright E2E tests, that Theme related tests are not muted.

Local
- pull changes
- transpile with `yarn workspace @automattic/calypso-e2e build --watch` 
- run the following:
```
DEBUG=pw:api mocha --config .mocharc_playwright.yml specs/specs-playwright/wp-theme__switch-activate-spec.js
```

Blocked by #54006
